xml/abstract_commands.xml

```diff
@@ -191,6 +191,10 @@
 
             1: Addresses are virtual, and translated the way they would be from
             M-mode, with \FcsrMcontrolMprv set.
+
+            Debug Modules on systems without address translation (i.e. virtual addresses equal physical)
+            may optionally allow \FacAccessmemoryAamvirtual bit set to 1, which would produce the same result as
+            the same abstract command with \FacAccessmemoryAamvirtual bit cleared.
```

Comment:
Our convention seems to be to say just "\FacAccessmemoryAamvirtual" instead of "\FacAccessmemoryAamvirtual bit". Not all the spec does that, but most does, and it is more concise while just as readable.

Reply:
Thanks, I have reworded it as suggested.